chips/nrf52/src/uart.rs

```diff
@@ -121,6 +120,10 @@ impl UART {
         let regs = unsafe { &*self.regs };
         let tx = regs.event_endtx.get() != 0;
 
+        if regs.event_error.get() == 1 {
+            panic!("uart error event\r\n");
```

Comment:
is this a temporary `panic!` until proper error handling is implemented? or is a uart error so unexpected here that it warrants stopping/restarting the entire system?